docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -207,47 +208,98 @@ function processMessage(arg) {
 
 ## Pass information to the dialog box
 
-Sometimes the host page needs to pass information to the dialog box. You can do this in two primary ways:
+Your add-in can send messages from the [host page](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page) to a dialog box using the [messageChild](/javascript/api/office/office.dialog#messagechild-message-) method of the [Dialog](/javascript/api/office/office.dialog) object.
 
-- Add query parameters to the URL that is passed to `displayDialogAsync`.
-- Store the information somewhere that is accessible to both the host window and dialog box. The two windows do not share a common session storage, but *if they have the same domain* (including port number, if any), they share a common [Local Storage](https://www.w3schools.com/html/html5_webstorage.asp).\*
+> [!Note]
+> The APIs are supported in only Excel, PowerPoint, and Word. Support for Outlook is coming soon.
```

Comment:
Should we hedge the Outlook support as "in development" instead of "coming soon"? I'm not sure about the connotations of each, but I'm worried about customer comments like "This has been 'coming soon' for X months."

Reply:
OK